### ISSUE TITLE
Decide on the number of views for Data Entry

### ISSUE THREAD

**Issue by Pr1sM** [Swartdogs/PowerScout#23]
I haven't seen any discussion over this topic yet, but this needs to be decided before we can proceed with the data entry portion of the app: How many views do we want to have for data entry? In the past we've had an initial view for entering basic info (team number, match number, alliance), a view for data entry (pre-game, auto, teleop), and a final view for post-game results (final score, penalties, etc.).  

I see from the DataEntry storyboard that we have attempted to compress all of these into one view. A couple of quick thoughts on this (feel free to respond below):
- PRO: Having one view is simpler
- PRO: Use the full iPad screen, won't potentially waste screen space
- CON: No early exit condition for no shows
- CON: More controls per page, scouters may miss things when entering data
- CON: Less space per control (or short description accompanying control)
- CON: Direct dependency on iPad screen, multiple views could allow use on iPhones

Overall, I think having more views could also help for future games as well. We know that the basic info will always be something we want to track (it identifies the match) and we know that post-game results will always be something we want to track (it shows a summary of the match). If these things are going to stay from year to year, putting them in separate views means they can be recycled for future years and we only have to change one view (game specific metrics). 

What are your thoughts on this? I'm sure there are more pros than what I listed above! We need to figure this out before we go much further with the data entry side of things. We are reaching a point where the last thing (besides bugs!) is creating and saving matches, which rely on a storyboard of one or more views.

**Comment by Pr1sM** [comment]
For now, I think we will be going with a 3 view data entry flow: One for Basic Team Info, one for Game Specific Data Entry, and one for Final Results. If we decide to change this, a new issue can be made or this one can be reopened.
